docs/dev-docker/README.md

```diff
@@ -95,21 +91,21 @@ The table below shows latency measurement, which typically involves assessing th
 
 *TP stands for Tensor Parallelism.*
 
-Supermicro AS-8125GS-TNMR2 with 2x AMD EPYC 9554 Processors, 2.25 TiB RAM, 8x AMD Instinct MI300X (192GiB, 750W) GPUs, Ubuntu 22.04, and amdgpu driver 6.8.5
+Supermicro AS-8125GS-TNMR2 with 2x AMD EPYC 9554 processors, 2.25 TiB RAM, 8x AMD Instinct MI300X (192GiB, 750W) GPUs, Ubuntu 22.04, and amdgpu driver 6.8.5
 
-## Reproducing Benchmarked Results
+## Reproducing benchmark results
 
-### Preparation - Obtaining access to models
+### Preparation- accessing the models
 
-The vllm docker image should work with any model supported by vLLM.  When running with FP8, AMD has quantized models available for a variety of popular models, or you can quantize models yourself using Quark.  If needed, the vLLM benchmark scripts will automatically download models and then store them in a Hugging Face cache directory for reuse in future tests. Alternatively, you can choose to download the model to the cache (or to another directory on the system) in advance.
+The vLLM Docker image supports any model compatible with vLLM.  When running with FP8,AMD provides quantized versions of several popular models, or you can quantize models yourself using Quark.  The vLLM benchmark scripts can automatically download models and store them in a Hugging Face cache directory for reuse in future tests. Alternatively, you can pre-download the model to the cache or another directory on your system.
 
-Many HuggingFace models, including Llama-3.1, have gated access.  You will need to set up an account at (https://huggingface.co), search for the model of interest, and request access if necessary. You will also need to create a token for accessing these models from vLLM: open your user profile (https://huggingface.co/settings/profile), select "Access Tokens", press "+ Create New Token", and create a new Read token.
+Many HuggingFace models, including Llama-3.1, have gated access.  You will need to set up an account at [Hugging Face](https://huggingface.co), search for the model of interest, and request access if necessary. You will also need to create a token for accessing these models from vLLM by going to your [user profile](https://huggingface.co/settings/profile), selecting **Access Tokens**, clicking **+ Create New Token**, and creating a new **Read** token.
 
 ### System optimization
 
-Before running performance tests you should ensure the system is optimized according to the [ROCm Documentation](https://rocm.docs.amd.com/en/latest/how-to/system-optimization/mi300x.html).  In particular, it is important to ensure that NUMA auto-balancing is disabled.
+Before running performance tests you should ensure the system is optimized according to the [ROCm documentation](https://rocm.docs.amd.com/en/latest/how-to/system-optimization/mi300x.html).  In particular, it is important to ensure that NUMA auto-balancing is disabled.
```

Comment:
Let's call it the ROCm MI300X System Optimization Guide explicitly for the hyperlink